package.json

```diff
@@ -111,6 +110,7 @@
     "throw-max-listeners-error": "1.0.1",
     "ua-parser-js": "0.7.24",
     "watch-glob": "0.1.3",
+    "whatwg-fetch": "3.6.2",
```

Comment:
@lucidNTR can you fix the order here?  I think `wd` should come before `whatwg-fetch`, and `npm` changes the order if I install/remove other deps.